survival_lib.py

```
@@ -560,18 +587,19 @@ def create_forest_plot_cox(res_df):
-    Returns:
-        html_string
     """
     if res_df is None or res_df.empty:
         return "<p>No Cox regression results available for forest plot.</p>"
     
-    fig = create_forest_plot_cox(res_df)
-    plot_html = fig.to_html(include_plotlyjs=True, div_id='cox_forest_plot')
+    try:
+        fig = create_forest_plot_cox(res_df)
+        plot_html = fig.to_html(include_plotlyjs=True, div_id='cox_forest_plot')
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)